rose/client/main.py

```diff
@@ -74,6 +74,14 @@ def main():
         d = {}
         exec f in d, d
 
-    reactor.connectTCP(d['server_address'], config.game_port,
+    if len(sys.argv) == 3:
+        server_address = sys.argv[2]
```

Comment:
This is not a good practice for optional argument. In this case we need a proper option:

    rose-client --server-address example.com my_driver.py

Reply:
I agree that argparse is probably the best solution for this.